test/odbc/psqlodbc/psqlodbc_tests_common.h

```diff
@@ -0,0 +1,337 @@
+#ifndef PSQLODBC_TESTS_COMMON_H
+#define PSQLODBC_TESTS_COMMON_H
+
+#include <cmath>
+#include <gtest/gtest.h>
+#include <sqlext.h>
+#include <string>
+#include <time.h>
+#include <vector>
+
+#include "../src/drivers.h"
+#include "../src/odbc_handler.h"
+#include "../src/query_generator.h"
+
+using std::vector;
+using std::string;
+using std::pair;
+
+const int BUFFER_LENGTH = 8192;
+const int BUFFER_SIZE = 256;
```

Comment:
Is it important/needed to have two generic BUFFER_LENGTH and BUFFER_SIZE?
Could the bigger one cover them both? 
Or if BUFFER_SIZE is needed for specific cases, would a better name indicate these?